Style.css

```diff
@@ -0,0 +1,146 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.top-bar {
+  display: flex;
+  width: 100%;
+  height: 50px;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logoname {
+  width: 100px;
+  height: 20px;
+  margin-left: 20px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+img {
+  margin-right: 20px;
+}
+
+.section-data {
+  background-image: url("./images/background.png");
```

Comment:
The background image should cover the whole section but in your case, it just covers a fraction of it. Kindly fix this issue.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/219361079-cf606330-42a5-4fc9-903b-72c1be2225b0.png) | ![image](https://user-images.githubusercontent.com/33268927/219360988-83c2a84f-b414-49f3-b777-2d1f7e951b42.png) |

Reply:
- [ ] I think you need to assign `height` property for background image to cover the whole section. 👍  
```suggestion
  background-image: url("./images/background.png");
  height: 575px;
```